AnimatedTextInput/Classes/AnimatedTextInput.swift

```diff
@@ -23,7 +23,13 @@ open class AnimatedTextInput: UIControl {
             configureType()
         }
     }
-
+    
+    open var autocorrection: UITextAutocorrectionType = .no {
+        didSet {
+            configureType()
```

Comment:
Is this needed? `configureType()` removes the whole view from its superview and generates a new input. I would simply do `textInput.autocorrection = autocorrection`.

Reply:
I agree, you are right.
I made the proposed change and will do the commit soon.